src/kbmod/search/bindings.cpp

```diff
@@ -86,6 +86,7 @@ PYBIND11_MODULE(search, m) {
             .def("get_height", &ri::getHeight)
             .def("get_width", &ri::getWidth)
             .def("get_ppi", &ri::getPPI)
+            .def("get_all_pixels", &ri::getPixels)
```

Comment:
Documentation of any new C++ function is not a lot given the context, but it's better to do than not to do, even (or rather especially) when they're one-liners.